pkgs/tools/misc/yubikey-neo-manager/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, python27Packages, pyside, pycrypto
+, libykneomgr, yubikey-personalization, libu2f-host }:
+
+python27Packages.buildPythonPackage rec {
+  namePrefix = "";
+  name = "yubikey-neo-manager-${version}";
+  version = "1.4.0";
+  src = fetchurl {
+    url = "https://developers.yubico.com/yubikey-neo-manager/Releases/${name}.tar.gz";
+    sha256 = "1isxvx27hk0avxwgwcwys2z8ickfs816ii1aklvmi09ak1rgrf1g";
+  };
+
+  propagatedBuildInputs = [ pyside pycrypto ];
+  patches = [ ./fix-pyside-requirement.diff ];
+
+  # aid ctypes load_libary()
+  makeWrapperArgs = [
+    "--set LD_PRELOAD '${libykneomgr}/lib/libykneomgr.so ${yubikey-personalization}/lib/libykpers-1.so ${libu2f-host}/lib/libu2f-host.so'"
+  ];
```

Comment:
I had the impression that all store paths in the final derivation was automatically added as a runtime dependency. Is that not the case for wrappers? It seems to be added as a dependency, unless I'm misunderstanding something:

```
> nix-store -q --references /nix/store/7mayqdcjcjazv1i2cx5y536mvpf0jvlb-yubikey-neo-manager-1.4.0/bin/neoman
/nix/store/h2fqzsg1xzpbh05bkx3i3ygi3gcy1dm5-bash-4.3-p42
/nix/store/zpczhw3sg2kn7yksr2qz64sfpcy01v3p-python-2.7.11
/nix/store/wzgvn6mrvk927awf1hl0gkhn6w5zr7q2-python2.7-setuptools-19.4
/nix/store/22mivp2ca7nh43gw9fhg7hx8nsjgbgzz-pycrypto-2.6.1
/nix/store/6pcf406k8i4bhiyhc6n6c04r98gkcgl5-yubikey-personalization-1.17.3
/nix/store/6yfc1yyvpm90vkfl7xxzha9i2ngmy89c-libykneomgr-0.1.8
/nix/store/75hgpvrjq6j7xb1syc67jv1zqgb2dbd7-libu2f-host-1.1.1
/nix/store/i9ffhl1pd5vs8xf2xb3mskqkfzis7f9y-python2.7-pyside-1.2.4
/nix/store/7mayqdcjcjazv1i2cx5y536mvpf0jvlb-yubikey-neo-manager-1.4.0
```